### DISCUSSION TITLE
Question about function name in ElasticDriver

### DISCUSSION THREAD

**Discussion by solin319** [horovod/horovod#2782] [Q&A]
**Environment:**
3. Horovod version: Maser

**Your question:**
The funcion register_worker_server in class ElasticDriver, used to regist worker client.
Why not change the  following function name to register_worker_client.

![image](https://user-images.githubusercontent.com/13029886/107343174-0d289b80-6afc-11eb-96c9-da82791f4a7c.png)

**Comment by tgaddair** [comment]
Hey @solin319, I agree the function name is confusing.  I think something more appropriate might be "connect_worker_server".  The idea is that we want to establish a direct connection to the server running on the worker nodes, which we do by creating a client that can send it requests.

Feel free to create a PR if you would like!
